style.css

```diff
@@ -0,0 +1,70 @@
+
+nav {
+    margin-left: 25px;
+    margin-top:25px;
+    display: flex;
+}
+nav input{
+    height: 30px;
+    width: 500px;
+    margin-left: 125px;
+    margin-top:35px;
+}
+nav button {
+    height: 30px;
+    width: 40px; 
+    margin-left:-4px;
+    margin-top: 35px;
+}
+
+#add-video-playlist {
+    margin-left: 125px;
+    margin-top:38px; 
+}
+
+#bell{
+    margin-left: 70px;
+    margin-top:38px;
+}
+
+#profile-pic img{
+    margin-left: 70px;
+    margin-top:38px;
+    height: 50px;
+    width: 50px;
+    border-radius: 50%;
+}
+
+section {
+    margin-left: 95px;
+    margin-top:55px;
+}
+#profile-record{
+    display: flex;
+}
+#video-title{
+    margin-top: 15px;
+}
+#like{
+    margin-top: 15px;
+    margin-left: 25px;
+}
+#dislike{
+    margin-top: 15px;
+    margin-left: 15px;
+}
+#share{
+    margin-top: 27px;
+    margin-left: 25px;
+}
+#subscribe{
+    margin-top: 27px;
+    margin-left: 25px;
+}
+#review-info{
+    display: flex;
+}
```

Comment:
- [ ] For good coding practices, please try to use classes instead of id's, since they can be reused and help to optimize your CSS code